### ISSUE TITLE
Discussion : Warnings on imports

### ISSUE THREAD

**Issue by zekth** [denoland/deno#2233]
My concern is about imports from URL and especially when you import from CDN / package management. If you don't use a tagged version you'll import the latest one, which is not recommended in production environment for example.

Example of my experience:
Several teams are using `Docker` and multiple images in their workflow. Sometimes the reviewer don't check this part of the PR or commits due to workload and so on (i include myself too). Then sometimes we got some incompatibilities and it breaks.

In conclusion i think it would be nice to add a warning like `Importing non tagged package` in the compiler. Added to this having an option like in `eslint` `--max-warnings=0` will prevent this behaviour.

**Comment by hayd** [comment]
Perhaps this could be in a linter be rather than compiler warning?

When there is dependencies as json you could recursively loop through every dep url and assert it's a tagged url.

**Comment by zekth** [comment]
I like this idea. Also it could check multiple versions of the same package in deps.

**Comment by ry** [comment]
Deno only sees URLs, which don't have a concept of a tag. So there's no way to make this warning. As suggested, maybe a linter could inspect URLs from certain domains.
